README.md

```diff
@@ -88,6 +87,11 @@ to use, test, and provide feedback on the new features are encouraged to try
 this branch. However, this branch provides no stability guarantees and reserves
 the right to change and break the API at any time.
 
+##Prerequisite knowledge
```

Comment:
While this may be a valid markdown, it isn't displayed correctly on github,
A space is missing.